docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       - DESEC_NS_NAME
     volumes:
       - certs:/etc/certs
-      - ./certs/entrypoint.sh:/usr/local/bin/entrypoint.sh
+      - ./dox-certs/entrypoint.sh:/usr/local/bin/entrypoint.sh
```

Comment:
I think we should keep the service name and the directory name in sync.

Reply:
done